pkgs/development/python-modules/gin-config/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, six
+, enum34
+, absl-py
+, tensorflow
+, withTensorflow ? true
```

Comment:
This is pointless. If one (does not) want Tensorflow, they will need to override this expression. In that case they might as well use `overridePythonAttrs`. Tensorflow is an optional dependency, so if someone wants to use this package in conjunction with tensorflow, they can just add both to the environment.

Reply:
you're right, sorry, missed that during my review :/

Reply:
I made changes to the derivation based on what I _think_ you mean, hope it's correct!